Segment/Classes/SEGSegmentIntegration.m

```diff
@@ -30,6 +30,12 @@
 NSString *const kSEGQueueFilename = @"segmentio.queue.plist";
 NSString *const kSEGTraitsFilename = @"segmentio.traits.plist";
 
+#if TARGET_OS_IPHONE
```

Comment:
I missing why this is needed, is it to avoid tvOS?

Reply:
It's not.  fixed.